File: cpp/ProdukElektronik.cpp
#include <iostream>
#include <string>
using namespace std;

class ProdukElektronik {
private:
    int id;
    string nama_produk;
    string deskripsi;
    int watt;
    double harga;
    string model;
    string merek;

public:
    // Konstruktor default
    ProdukElektronik() {
        id = 0;
        nama_produk = "";
        deskripsi = "";
        watt = 0;
        harga = 0.0;
        model = "";
        merek = "";
    }

    // Konstruktor dengan parameter
    ProdukElektronik(int id, string nama, string desk, int watt, double harga, string model, string merek) {
        this->id = id;
        this->nama_produk = nama;
        this->deskripsi = desk;
        this->watt = watt;
        this->harga = harga;
        this->model = model;
        this->merek = merek;
    }

    int getId() {
        return id;
    }

    void tampilkan() {
        cout << "ID Produk    : " << id << endl;
        cout << "Nama Produk  : " << nama_produk << endl;
        cout << "Deskripsi    : " << deskripsi << endl;
        cout << "Watt         : " << watt << " W" << endl;
        cout << "Harga        : Rp " << harga << endl;
        cout << "Model        : " << model << endl;
        cout << "Merek        : " << merek << endl;
        cout << "-----------------------------" << endl;
    }
};

File: java/ProdukElektronik.java
public class ProdukElektronik {
    private int id;
    private String namaProduk;
    private String deskripsi;
    private int watt;
    private double harga;
    private String model;
    private String merek;

    public ProdukElektronik(int id, String namaProduk, String deskripsi,
                            int watt, double harga, String model, String merek) {
        this.id = id;
        this.namaProduk = namaProduk;
        this.deskripsi = deskripsi;
        this.watt = watt;
        this.harga = harga;
        this.model = model;
        this.merek = merek;
    }

    public int getId() {
        return id;
    }

    public void tampilkan() {
        System.out.println("ID Produk   : " + id);
        System.out.println("Nama Produk : " + namaProduk);
        System.out.println("Deskripsi   : " + deskripsi);
        System.out.println("Watt        : " + watt + " W");
        System.out.println("Harga       : Rp " + harga);
        System.out.println("Model       : " + model);
        System.out.println("Merek       : " + merek);
        System.out.println("-------------------------------");
    }
}
